README.md

```diff
@@ -0,0 +1,103 @@
+## Generate a new Maven project ##
+
+- Run below command from Command line tool
+
+```text
+mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4
```

Comment:
<div>
<h3>Code Review Agent Run <a href = https://github.com/Amruta101998/selenium-automation/pull/2#issuecomment-2123048177>#0a0bf</a> - 05/21/2024, 10:25 pm</h3>

<p>🔴 High importance</p>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The Maven command provided for generating a new project is inefficient for large-scale projects as it uses a non-interactive mode which might not utilize caching mechanisms effectively.
</div>
<div id="fix">
<b>Fix:</b> Modify the Maven command to enable interactive mode and caching. This can significantly improve the build performance by reusing previously downloaded artifacts and reducing the time spent on network operations.
</div>
<details>
<summary>
Code suggestion
</summary>
<div>
<pre><code>
 @@ -6,1 +6,1 @@
 -mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4
 +mvn archetype:generate -DinteractiveMode=false -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DarchetypeCatalog=local
</code></pre>
</div>
</details>
<h4><a href="https://github.com/Amruta101998/selenium-automation/pull/2#issuecomment-2123048177"><-- Code Review Agent Run#0a0bf</a></h4>
<i>AI Code Review powered by <img src="https://bito.ai/wp-content/uploads/2023/10/Logo-Bito-Black-cropped.svg" alt="Bito Logo" width="50" height="25"></i>
</div>
<br>

<br>

</div>